app/vmalert/manager.go

```diff
@@ -26,8 +26,9 @@ type manager struct {
 	wg     sync.WaitGroup
 	labels map[string]string
 
-	groupsMu sync.RWMutex
-	groups   map[uint64]*Group
+	groupsMu        sync.RWMutex
+	groups          map[uint64]*Group
+	groupsReloadErr error
```

Comment:
Could this field be a global variabe? Is there a reason why it should belong to manager object?

Reply:
used global variabel and renamed it to userError